libs/common/map_descriptors.cpp

```diff
@@ -0,0 +1,101 @@
+/*
+ *  Copyright (c) Microsoft Corporation
+ *  SPDX-License-Identifier: MIT
+ */
+
+#include <vector>
+#include "api_common.hpp"
+
+// TODO: this duplicates global_program_info.map_descriptors in ebpfverifier.lib
+// https://github.com/vbpf/ebpf-verifier/issues/113 tracks getting rid of global
+// state in that lib, but won't notice this global state which has the same
+// problem.
```

Comment:
https://github.com/vbpf/ebpf-verifier/issues/113 is now closed by making the global state be thread local.  At least the comment needs updating now to say this should be thread local